test/scripts/products/h24-verify-selfmanaged-apicast-and-custom-policy/test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+go run h24-verify-selfmanaged-apicast-and-custom-policy.go -apicast-image-tag=3scale2.10.0 \
+-admin-password=Password1 \
+-ocm-token=eyJhbGciOiJIUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJhZDUyMjdhMy1iY2ZkLTRjZjAtYTdiNi0zOTk4MzVhMDg1NjYifQ.eyJpYXQiOjE2MjU0NjQ5ODAsImp0aSI6IjI3MGJkZjQ0LWQ4N2EtNGI1Yi1hN2I3LWI3YjgzY2I4OTQ2MCIsImlzcyI6Imh0dHBzOi8vc3NvLnJlZGhhdC5jb20vYXV0aC9yZWFsbXMvcmVkaGF0LWV4dGVybmFsIiwiYXVkIjoiaHR0cHM6Ly9zc28ucmVkaGF0LmNvbS9hdXRoL3JlYWxtcy9yZWRoYXQtZXh0ZXJuYWwiLCJzdWIiOiJmOjUyOGQ3NmZmLWY3MDgtNDNlZC04Y2Q1LWZlMTZmNGZlMGNlNjp2bW9naWxldl9yaG1pIiwidHlwIjoiT2ZmbGluZSIsImF6cCI6ImNsb3VkLXNlcnZpY2VzIiwibm9uY2UiOiIxMDg5Nzc0ZC0zYzU0LTQ1MTctOGUzMS04OGNlZTVmNWI5ZDYiLCJzZXNzaW9uX3N0YXRlIjoiOWZiZDVjOGQtZGNiMi00MDdiLWEyY2QtZDQ5ODI4MDM5N2M0Iiwic2NvcGUiOiJvcGVuaWQgb2ZmbGluZV9hY2Nlc3MifQ.anSM5v5DBeAWDz6L27-BkYWI5ieCnfUVQ7GzURiRT7o
```

Comment:
Hi Pavel, Thank you very much for your Review. Yes, sure, I agree, but it depends also who will use this test,  I thought that this script is for internal testing. Maybe secret somehow could be used. If you think that some preparation could be test, it's ok, Just maybe we could add some checking of prerequisites, so it will be easier to user, no need look at documentation. Thank you very much for this comment. I will look how to improve.

Reply:
Thanks Valery, I would suggest to remove the token as soon as possible from this PR, because this repository is public.

Reply:
Done. Thanks a lot! sorry ... I thought that token is temporary.

Reply:
Hi Pavel. I commented meanwhile OCM_LOGIN() function that using Token. If we don't need login to OCM, - Token can be removed. But if we need ocm login - I'd like suggeste request input from command  line, so it will be more secure.  
Regarding customer-admin password, maybe we also could request input of password from command line, and we will not print it. What do you think? 
Thank you.

Reply:
Refactoring done.  Tokens/passwords - removed from ./test.sh. All required authentication details, where required, will be request by script and entered in command prompt; and should not be printed.